e2e-tests/tests.sh

```diff
@@ -12,6 +12,16 @@ DUMMY_DEPLOYMENT_NAME="dummy-deployment"
 DUMMY_POD_LABEL="app=${DUMMY_DEPLOYMENT_NAME}"
 ENV_VARS_PREFIX="NEW_RELIC_METADATA_KUBERNETES"
 NAMESPACE_NAME="$(kubectl config view --minify --output 'jsonpath={..namespace}')"
+IMAGE_TAG="e2e-test"
+
```

Comment:
Same comment as above: please use the notation we use in other repos
https://github.com/newrelic/nri-kube-events/blob/main/.github/workflows/e2e.yaml#L42